docs/chapters/sde/index.rst

```diff
@@ -0,0 +1,14 @@
+.. include:: ../../swf-blueprint/docs/articles/sde/introduction_to_sde.rst
```

Comment:
Is it possible to restructure with a toctree of the intro and the Download instructions? includes tend to mess with references etc.

Reply:
I tried it and could not get a nice solution...